src/Aspire.Dashboard/wwwroot/js/app.js

```diff
@@ -114,8 +128,45 @@ function isScrolledToBottom(container) {
     return difference < marginOfError;
 }
 
+function copyUrlContentToClipboard(url) {
+    // code from https://wolfgangrittner.dev/how-to-use-clipboard-api-in-firefox/
+    if (typeof ClipboardItem && navigator.clipboard.write) {
+        // NOTE: Safari locks down the clipboard API to only work when triggered
+        //   by a direct user interaction. You can't use it async in a promise.
+        //   But! You can wrap the promise in a ClipboardItem, and give that to
+        //   the clipboard API.
+        //   Found this on https://developer.apple.com/forums/thread/691873
+        const text = new ClipboardItem({
+            "text/plain": fetch(url)
+                .then((response) => response.text())
+                .then((text) => new Blob([text], { type: "text/plain" }))
```

Comment:
Does this work on Safari? It would make using the clipboard easier in a lot of situations.

Reply:
Hmm, I was hoping we could combine this with the technique shown here: https://learn.microsoft.com/en-us/aspnet/core/blazor/file-downloads?view=aspnetcore-8.0#download-from-a-stream

But it requires a roundtrip to the server to get the stream reference.